utils/install_v2.sh

```diff
@@ -593,11 +593,23 @@ main() {
 			[ -f "$__HOME__/.zprofile" ] && echo "$_source" >>"$__HOME__/.zprofile"
 		fi
 	elif [[ "$_shell_" =~ "bash" ]]; then
-		local _grep=$(cat "$__HOME__/.bash_profile" 2>/dev/null | grep "$IPATH/env")
-		if [ "$_grep" = "" ]; then
-			# If the .bash_profile is not existing, create a new one
-			[ ! -f "$__HOME__/.bash_profile" ] && touch "$__HOME__/.bash_profile"
-			[ -f "$__HOME__/.bash_profile" ] && echo "$_source" >>"$__HOME__/.bash_profile"
+		if [ -f "$__HOME__/.bash_profile" ]; then
+			# If .bash_profile already exists, use it (user intentionally created it)
+			local _grep=$(cat "$__HOME__/.bash_profile" 2>/dev/null | grep "$IPATH/env")
+			if [ "$_grep" = "" ]; then
+				echo "$_source" >> "$__HOME__/.bash_profile"
+			fi
+		else
+			# If .bash_profile doesn't exist, check if .profile exists
+			if [ -f "$__HOME__/.profile" ]; then
+				local _grep=$(cat "$__HOME__/.profile" 2>/dev/null | grep "$IPATH/env")
+				if [ "$_grep" = "" ]; then
+					echo "$_source" >> "$__HOME__/.profile"
+				fi
+			else
+				# neither exists, create .profile (Ubuntu convention)
```

Comment:
Please note that this script is not only for Ubuntu. It runs on many distributions such as Debian, Fedora, macOS, and RHEL. Creating a `.profile` by default to comply with the Ubuntu convention may not work on other operating systems.